hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/helpers/BlockDeletingServiceMetrics.java

```diff
@@ -74,6 +74,10 @@ public final class BlockDeletingServiceMetrics {
   @Metric(about = "The total number of Container chosen to be deleted.")
   private MutableGaugeLong totalContainerChosenCount;
 
+  @Metric(about = "The total number of transactions which was failed " +
```

Comment:
which was failed cause by wait Container lock timeout -> which failed due to container lock wait timeout

Reply:
Done.